packages/dashboard/package.json

```
@@ -6,30 +6,25 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
-    "do-lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint .",
```

```suggestion
    "do-lint": "eslint .",
```
I need the `do-lint` command for pre-commit hooks.